Mathlib/LinearAlgebra/RootSystem/Basic.lean

```diff
@@ -113,23 +129,25 @@ lemma reflection_dualMap_eq_coreflection :
   ext n m
   simp [coreflection_apply, reflection_apply, mul_comm (P.toLin m (P.coroot i))]
 
-variable [Finite ι]
+lemma isCrystallographic_iff :
+    P.IsCrystallographic ↔ ∀ i j, ∃ (z : ℤ), z = P.pairing i j := by
```

Comment:
```suggestion
    P.IsCrystallographic ↔ ∀ i j, ∃ z : ℤ, z = P.pairing i j := by
```
(wasn't there a discussion about `∃ x, x = ...` vs `∃ x, ... = x` recently? I can't find it now, but maybe check if this matches the conclusion, if you can find it)